native/libcst/src/nodes/statement.rs

```diff
@@ -1713,6 +1715,68 @@ impl<'a> Inflate<'a> for ExceptHandler<'a> {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, IntoPy)]
+pub struct ExceptStarHandler<'a> {
+    pub body: Suite<'a>,
+    pub r#type: Expression<'a>,
+    pub name: Option<AsName<'a>>,
+    pub leading_lines: Vec<EmptyLine<'a>>,
+    pub whitespace_after_except: SimpleWhitespace<'a>,
+    pub whitespace_after_star: SimpleWhitespace<'a>,
+    pub whitespace_before_colon: SimpleWhitespace<'a>,
+
+    pub(crate) except_tok: TokenRef<'a>,
+    pub(crate) star_tok: TokenRef<'a>,
+    pub(crate) colon_tok: TokenRef<'a>,
+}
+
+impl<'a> Codegen<'a> for ExceptStarHandler<'a> {
+    fn codegen(&self, state: &mut CodegenState<'a>) {
+        for ll in &self.leading_lines {
+            ll.codegen(state);
+        }
+        state.add_indent();
+
+        state.add_token("except");
+        self.whitespace_after_except.codegen(state);
+        state.add_token("*");
+        self.whitespace_after_star.codegen(state);
+        self.r#type.codegen(state);
+        if let Some(n) = &self.name {
+            n.codegen(state);
+        }
+        self.whitespace_before_colon.codegen(state);
+        state.add_token(":");
+        self.body.codegen(state);
+    }
+}
+
+impl<'a> Inflate<'a> for ExceptStarHandler<'a> {
+    fn inflate(mut self, config: &Config<'a>) -> Result<Self> {
```

Comment:
I didn't read enough of the original Rust parser PR to understand `Inflate`. What does it do, and what is `Config`? From the code it seems related to parsing, but that's what the parser does.

Is this a post-processing step after the parser, where we basically move whitespace up from child CST structs to the parent CST structs? At a skim that kind of seems to be what's going on here, e.g. reassigning whitespace from the except token to self.

Reply:
> Is this a post-processing step after the parser, where we basically move whitespace up from child CST structs to the parent CST structs

Yes, exactly. `Config` is all the data whitespace parsing needs (e.g. default indentation)